prboom2/src/m_cheat.c

```diff
@@ -133,6 +136,7 @@ cheatseq_t cheat[] = {
   CHEAT("idbeholdl",  "Lite-Amp Goggles", not_dm, cheat_pw, pw_infrared),
   CHEAT("idbehold",   "BEHOLD menu",      not_dm, cheat_behold, 0),
   CHEAT("idclev",     "Level Warp",       cht_never | not_menu, cheat_clev, -2),
+  CHEAT("idclev",     NULL,               cht_never | not_menu, cheat_clev0, 0),
```

Comment:
I am not sure if this should read "Level Warp" as well? Else, if the cheat sequence is replaced by DEHACKED, it will print the level information when IDCLEV is typed but won't warp to the entered level and if the replacement cheat sequence for IDCLEV is typed it will warp without showing this information at all.